### PR TITLE
Security improvements

### PR DESCRIPTION
**Pull request by npalm**
Solve #4  and #12

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -207,3 +207,9 @@ variable "tags" {
   description = "A map of tags to add to the resources"
   default     = {}
 }
+
+variable "ssl_policy" {
+  type        = "string"
+  description = "SSL policy applied to an SSL enabled ALB, see https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-security-policy-table.html"
+  default     = "ELBSecurityPolicy-2015-05"
```
Comment (JeroenKnoops) [inline variables.tf]: Why do we set this as default and not the recommended: ELBSecurityPolicy-2016-08 https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-security-policy-table.html

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
+- Added: SSL policy is configurable
+- Changed: Removed egres from alb ecurity group, limit port to alb port only.
```
Comment (JeroenKnoops) [inline CHANGELOG.md:9]: **security group** instead of **ecurity group**

**Review by JeroenKnoops — COMMENTED** [review]
Small notes

File: variables.tf (hunk 1 above)
Comment (npalm) [reply variables.tf]: Update the default to the latest version, add comment to changelog.

**Review by JeroenKnoops — APPROVED** [review]
THanks! Great Work!

**Pull request merged**